pkg/manager/impl/execution_manager.go

```diff
@@ -619,6 +619,12 @@ func (m *ExecutionManager) launchExecutionAndPrepareModel(
 		return nil, nil, err
 	}
 
+	projectLabels := make(map[string]string)
+	project, err := m.db.ProjectRepo().Get(ctx, request.Project)
+	if err != nil {
```

Comment:
if the err is not nil, the project definitely won't be populated. we should bubble up the err and return in this case

Reply:
addressed